new/MDDL003.xml

```diff
@@ -0,0 +1,386 @@
+                        <msItem xml:id="ms_i6">
+                            <locus from="85vb" to="86ra"/> 
+                            <title type="complete" ref="LIT3313Severus"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i7">
+                            <locus from="86ra" to="86vc"/> 
+                            <title type="complete" ref="LIT3314Acacius"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i8">
+                            <locus from="86vc" to="87rb"/> 
+                            <title type="complete" ref="LIT3315Juvenal"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i9">
+                            <locus from="87rb" to="88ra"/> 
+                            <title type="complete" ref="LIT3331Cyril"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i10">
+                            <locus from="88ra" to="88rc"/> 
+                            <title type="complete" ref="LIT3316Rheginus"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i11">
+                            <locus from="88rc" to="89ra"/> 
+                            <title type="complete" ref="LIT3331Cyril"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i12">
+                            <locus from="89ra" to="89rc"/> 
+                            <title type="complete" ref="LIT3317Eusebius"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i13">
+                            <locus from="89rc" to="90ra"/> 
+                            <title type="complete" ref="LIT3318Theodo"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i14">
+                            <locus from="90ra" to="90rb"/> 
+                            <title type="complete" ref="LIT3319Firmus"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i15">
+                            <locus from="90rb" to="90va"/> 
+                            <title type="complete" ref="LIT3320Letter"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i16">
+                            <locus from="90va"/> 
+                            <title type="complete" ref="LIT3331Cyril"/>
```

Comment:
does it really have the same text twice (i11 and i16)?

Reply:
I corrected it.  it was misleading by EMIP00641.